pkg/sql/catalog/typedesc/table_implicit_record_type.go

```diff
@@ -175,7 +175,6 @@ func (v TableImplicitRecordType) Adding() bool {
 
 // Dropped implements the Descriptor interface.
 func (v TableImplicitRecordType) Dropped() bool {
-	v.panicNotSupported("Dropped")
 	return false
```

Comment:
Shouldn't this instead delegate to the wrapped table descriptor?

Reply:
Thanks, done